content/References/dRAIDprimer.md

```diff
@@ -0,0 +1,115 @@
+If adding spares after pool creation is a critical requirement, create the pool using a raidz layout.
+
+**Sequential resilver**
+
+This is a new resilver type used by dRAID where blocks are read and restored sequentially.
+This greatly increases resilver operation speed as it does not require block tree traversal.
+
+Sequential resilver is made possible by the use of fixed stripe width in dRAID.
+During a sequential resilver data is read from all pool members.
+
+Checksums are not validated during a sequential resilver.
+However, a scrub begins after the sequential resilver finishes and verifies data checksums.
+
+**Fixed stripe width**
+
+Stripe width is fixed in dRAID, with zeros added as padding.
+This can have a significant effect on usable capacity, depending on the data stored.
+
+In a redundancy group of eight data disks using 4k sector disks, the minimum allocation size is **32k**.
+Any files smaller than 32k still fill an entire stripe, with zeros appended to the write to fill the entire stripe.
+This greatly reduces the pool usable capacity when the pool stores large numbers of small files.
+
+dRAID datasets benefit greatly from larger record sizes.
+**128k** is recommended as a minimum value for datasets and **64k** to **128k** for zvols.
+Selecting a record/block size smaller than the minimum allocation size is catastrophic for pool capacity.
+
+**Rebalancing**
+
+This process occurs after a disk failure results in a distributed hot spare replacing a failed drive.
+This is essentially a resilver, but data redistributes across the pool to meet the original distributed configuration.
+
+During rebalancing, a traditional resilver is performed as the pool is not in a degraded state.
+Checksums are also verified during this process.
+
+## Terminology
+
+The terminology used for a dRAID layout is similar to that used for raidz, but could have a different meaning.
+It is recommended to review this list of terms and definitions before attempting to create storage pools with a dRAID layout in TrueNAS:
+
+{{< truetable >}}
+| Term | Definition |
+|------|------------|
+| Children (**C**) | Number of drives included in the dRAID deployment.
+| Data Devices (**D**) | Number of data devices in a redundancy group. This number can be quite high, but generally a lower number results in greater performances and capacity is more effectively used. |
+| Distributed hot spare | Unlike in a raidz configuration where spares remain inactive until needed, in a dRAID configuration spare capacity is distributed across the drives. This results in all drives being active members of the pool.
+| Parity Level (**P**) | Distributed parity level of a dRAID redundancy group. This ranges from **1** to **3** and is similar to the RAIDz parity level. |
+| Redundancy group | Similar to a **vdev** in raidz. A redundancy group is composed of parity devices and data devices. Redundancy group size impacts storage performance and capacity. |
```

Comment:
This definition should be scrubbed to diferentiate it from Vdev below. Maybe just cut the first sentence--or explain in what way it's similar to vdevs in raidz